R/explorer.R

```diff
@@ -26,45 +49,58 @@ explorer <- function(dataList=c(), addEnv=T, demo=F) {
     )
   )
 
-  formatFileList<-function(fileList){
-    fileList_formatted = list()
-    i<-1
-    for(f in fileList){
-      df<-get(f)
-      if(class(df)=="data.frame"){
-        fileObj = list(
-          File=f,
-          Rows = dim(df)[1],
-          Columns = dim(df)[2],
+  formatFileList <- function(fileList){
```

Comment:
The "list" part of this function applies to when dataList is a list of data frames.

The "vector" part of this function applies to when dataList is a char vector, as well as working with the environment and demo datasets.